<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>785</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project Dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="HBoxProjView" stretch="0,1">
     <item>
      <widget class="QListWidget" name="ProjectView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="ShotTaskView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HBoxProjBtns">
     <item>
      <widget class="QPushButton" name="NewProjBtn">
       <property name="text">
        <string>New Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewShotBtn">
       <property name="text">
        <string>New Shot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewTaskBtn">
       <property name="text">
        <string>New Task</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="Line02">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HBoxLaunchBtns">
     <item>
      <layout class="QVBoxLayout" name="VBoxInfoLbls">
       <item>
        <widget class="QLabel" name="ProjNameLbl">
         <property name="text">
          <string>Project Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ResLbl">
         <property name="text">
          <string>Resolution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="FPSLbl">
         <property name="text">
          <string>FPS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VBoxSoftBtns">
       <item>
        <widget class="QPushButton" name="HouBtn">
         <property name="text">
          <string>Houdini</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NukBtn">
         <property name="text">
          <string>Nuke</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VBoxToolBtns">
       <item>
        <widget class="QPushButton" name="DeleteProjBtn">
         <property name="text">
          <string>Delete Project</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteShotTaskBtn">
         <property name="text">
          <string>Delete Shot / Task</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="Line01">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HBoxSettings" stretch="1,3,0,0">
     <item>
      <widget class="QLabel" name="RepoLocationLbl">
       <property name="text">
        <string>cc-pipe repo location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="VersionLbl">
       <property name="text">
        <string>cc-pipe version</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SettingsBtn">
       <property name="text">
        <string>settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBtn">
       <property name="text">
        <string>help</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
